README.md

```diff
@@ -58,6 +58,66 @@ Response:
 OK
 ```
 
+### `GET /miner/:id/deals/eligible/summary`
+
+Parameters:
+- `minerId` - a miner id like `f0814049`
```

Comment:
```suggestion
- `id` - a miner id like `f0814049`
```

Reply:
alternatively, change the URLs: `GET /miner/:minerId/...`

Reply:
I'll use `:minerId` to be explicit about what kind of ID we mean.